### PR TITLE
update links

### LINKED ISSUE #1: update links

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: collective/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# See http://peak.telecommunity.com/DevCenter/setuptools#namespace-packages
+# See https://peak.telecommunity.com/DevCenter/setuptools#namespace-packages
```
Comment (ale-rt) [inline collective/__init__.py]: The link is broken

File: setup.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
       long_description=open("README.txt").read() + "\n" +
                        open(os.path.join("docs", "HISTORY.txt")).read(),
       # Get more strings from
-      # http://pypi.python.org/pypi?%3Aaction=list_classifiers
+      # https://pypi.python.org/pypi?%3Aaction=list_classifiers
```
Comment (ale-rt) [inline setup.py]:
```suggestion
      # https://pypi.org/classifiers/
```

**Review by ale-rt — CHANGES_REQUESTED** [review]

File: collective/__init__.py (hunk 1 above)
Comment (Arhell) [reply collective/__init__.py]: Thank you, I will delete

File: setup.py (hunk 2 above)
Comment (Arhell) [reply setup.py]: Thank you

**Pull request merged**